css/lodesiempre.html

```diff
@@ -19,24 +17,168 @@
 
   <div class="header-background">
     <div class="column-container">
-        <h1 class="primary-font white-color margin-title">Apoya a los comercios gastronomicos.</h1>
-        <h3 class="secondary-font white-color margin-subtitle">Compra hoy un voucher con descuento, y ayuda a tu comercio favorito a surfear la ola.</h3>
+        <h1 class="primary-font white-color margin-title">Apoyá a los comercios gastronómicos.</h1>
+        <h4 class="secondary-font white-color margin-subtitle">Compra hoy un voucher con descuento, y ayudá a tu comercio favorito a surfear la ola.</h4>
         <h1 class="primary-font primary-color margin-title">417 personas</h1>
-        <h3 class="secondary-font white-color margin-subtitle"> ya apoyaron esta causa</h3>
-        <h2 class="primary-font primary-color margin-subtitle">80 restaurantes adheridos</h2>
+        <h4 class="secondary-font white-color margin-subtitle"> ya apoyaron esta causa</h4>
+        <h3 class="primary-font primary-color margin-subtitle">80 restaurantes adheridos</h3>
+    </div>
+  </div>
+
+  <div class="row-container">
```

Comment:
we can have a `<main>` tag that contains all the main content, and inside that multiple `<section>`s